packages/analytics/analytics-chart/src/utils/constants.ts

```diff
@@ -40,4 +42,8 @@ export function formatTime(ts: number, options: TimeFormatOptions = {}) {
   }
 }
 
+export function formatTimeRange(start: number, end: number) {
+  return `${formatTime(start)} - ${formatTime(end, { includeTZ: true })}`
+}
+
```

Comment:
Actually I think this belongs more in `analytics-utilities` package

Reply:
but `formatTime` is in `analytics-chart`, and it's imported from there in a few other places already.

And `formatTimeRange` requires `formatTime`, so I would have to duplicate it in `analytics-utilities`.  I will do so, to avoid breaking anything, leaving a comment in place in `analytics-chart` that we should eventually settle on `-utilities` as the source of truth.